VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -65,9 +79,28 @@ def device_list_for_billing(self):
         devices_to_bill = [] #list to store device names to be billed
         device_items = disk_util.get_device_items(None)
         for device_item in device_items :
-            # self.logger.log("Device name : {0} ".format(str(device_item.name)),True)
+            self.logger.log("Device name : {0} ".format(str(device_item.name)),True)
             if str(device_item.name).startswith("sd"):
                 devices_to_bill.append("/dev/{0}".format(str(device_item.name)))
+        self.logger.log("Initial billing items {0}".format(devices_to_bill))
+        self.logger.log("lines {0}".format(self.lines1))
+        for i in self.lines1:
+            lun=int(i[i.index(']')-1])
+            if lun not in self.IncludedLunList :
+                self.d=i.split()
+                #list containing the elements present in the row of the cmd sudo lsscsi
+                self.dvc=self.d[len(self.d)-1]#storing the corresponding device name
+                if lun not in self.dict1.keys():
+                    self.dict1[lun]=[]
+                self.dict1[lun].append(self.dvc)
+        self.logger.log("Lun numbers to be excluded {0}".format(self.dict1.keys()))
+        self.logger.log("Disks to be excluded {0}".format(self.dict1.values()))
+        for k in self.dict1.keys():
+            for v in self.dict1[k]:
+                for z in devices_to_bill:
```

Comment:
It might be better to take all values in self.dict1 into a hashtable and then loop items from devices_to_bill to check if it exists in the hashtable?